controllers/sriovoperatorconfig_controller.go

```diff
@@ -309,3 +324,65 @@ func (r *SriovOperatorConfigReconciler) syncK8sResource(ctx context.Context, cr
 	}
 	return nil
 }
+
+// syncSystemdService creates the Machine Config to deploy the systemd service on openshift ONLY
+func (r *SriovOperatorConfigReconciler) syncSystemdService(ctx context.Context, cr *sriovnetworkv1.SriovOperatorConfig) error {
```

Comment:
IMO, this method name should be OpenShift  specific, e.g.: syncOpenShiftSystemdService

Reply:
done